software-project-management-plan.md

```diff
@@ -6,17 +6,129 @@
+        - **Developers**
+      - **Planning Manager**
+      - **QA Manager**
+      - **Support Manager**
+    - **Sales Representative**
+    - **IT Department**
+
+#### External Stakeholders
+- **Client Company**
+- **Investors**
+- **Future External Companies**
+- **Technology Manufacturers**
+- **General Public**
+
 ### 2.3 Organisational Boundaries
+#### 2.3.1 Inclusive Boundaries
+
+- Support calls inside the client company.
+- Support audio and text.
+- Support major tech hardware and software.
+
+#### 2.3.2 Exclusive Boundaries
+
+- No integration with other calling software.
+- No support for calls outside the client company.
+- No support for video calls.
+
 ### 2.4 Project Responsibilities
+#### 2.4.1 Internal Roles and Responsibilities
+
+| Name & Organization | Project Role | Project Responsibilities |
+| ------------------- | ------------ | ------------------------ |
+| Sales Representative     | Project Team Member  | Market the product by communicating its purpose and features |
+| IT Department            | Project Team Member  | Support the onboarding of customers and requests from the developers |
+| Developers               | Project Team Member  | Engage in day-to-day activities, be on call when needed, technical writing, and contribute to project success |
+| Team Lead                | Project Team Member  | Monitor team progress, ensure the team stays on track, run work breakdowns, organize work assignments and provide updates to Project Manager |
+| Project Manager          | Project Team Member  | Ensure Team Lead is familiar with requirements, provide updates to Director/CEO, break tasks |
+| Director/CEO             | Project Sponsor      | Ensure Project Manager is up to date on the requirements and update Investors if needed |
+
+#### 2.4.2 External Roles and Responsibilities
+
+| Name & Organization      | Project Role                 | Project Responsibilities |
+| ------------------------ | ---------------------------- | ------------------------ |
+| Investors                | Secondary Stakeholder        | Provide financial support to the project |
+| Client Company           | Customer                     | Provide feedback and requests |
+| General Public           | Future Customer              | Offer future feedback and requests |
+| Future External Companies| Future Customer              | Offer future feedback and requests |
+| Technology Manufacturers | Hardware/software supporter  | Accommodate project requirements and support system changes |
+
+### 2.5 Risk Management Plan
+The SPMP shall specify:
+- Risk management plan for identifying, analyzing and prioritizing project risk factors.
+- Procedures for contingency planning and the methods that will be used for tracking certain risk factors, changes in levels of the factors and responses to those changes. The Synergy team will maintain then project’s risk factors and strategies for risk mitigation in a separate document, The U↑ Risk Management Plan.
```

Comment:
"synergy" -> U↑ :p

also i wonder if you can mention that this is included in the managerial plan? at the end of the process i plan on making a table of contents that links to each part, we can add that in afterwards

Reply:
I'm not sure what you mean. mention that what exactly is included in the managerial plan?

I updated the typo

Reply:
ah sorry i had meant that the risk management plan is included in the managerial plan section! no worries though, i'll push it through and we can just replace "…in a separate document, the U↑ Risk Management Plan." -> "…in the "Managerial Process" section of this document." or something like that